modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories/tails.dm

```diff
@@ -175,7 +191,6 @@
 	name = "Cat (Big)"
 	icon_state = "catbig"
 	color_src = USE_ONE_COLOR
```

Comment:
This one should probably be moved next to the `/datum/sprite_accessory/tails/mammal/wagging/tajaran` declaration, so it makes a bit more sense.